rib/rib_test.go

```diff
@@ -3043,6 +3043,110 @@ func TestRIBAddEntry(t *testing.T) {
 				},
 			},
 		}},
+	}, {
+		desc: "unresolved NHG",
+		inRIB: func() *RIB {
+			r := New(defName)
+			r.pendingEntries[1] = &pendingEntry{
+				ni: defName,
+				op: &spb.AFTOperation{
+					Id: 1,
+					Entry: &spb.AFTOperation_NextHopGroup{
+						NextHopGroup: &aftpb.Afts_NextHopGroupKey{
+							Id: 1,
+							NextHopGroup: &aftpb.Afts_NextHopGroup{
+								NextHop: []*aftpb.Afts_NextHopGroup_NextHopKey{{
+									// waiting on NH 42
+									Index:   42,
+									NextHop: &aftpb.Afts_NextHopGroup_NextHop{},
+								}},
+							},
+						},
+					},
+				},
+			}
+			return r
+		}(),
+		inNI: defName,
+		// install NH 1
+		inOp: &spb.AFTOperation{
```

Comment:
seems like this operation does not install NH1, it just install the same NHG#1 with NH#1 (with additional weight specified.)

Reply:
Agreed -- copy-pasta from a previous test. Cleaned this one and the below one up.